packages/nx-semantic-release/src/executors/semantic-release/semantic-release.ts

```diff
@@ -99,7 +103,7 @@ export async function semanticRelease(
 /**
  * @FIXME Recently semantic-release became esm only, but until NX will support plugins in ESM, we have to use this dirty hack :/
  * */
-function getSemanticRelease() {
+export/** for testing only!*/ function getSemanticRelease() {
```

Comment:
I have added a basic test for the `semnatic-release`executor,  but I was NOt able to make it work, specially because I didnt find a way to mock the ESM module...

Any help here is weelcome !

Reply:
Thanks for that! I will try to take a look at it when I will have a moment 😄